### ISSUE TITLE
Assign the GA4 Tracking ID in the Goodmarket

### ISSUE THREAD

**Issue by drptiago** [emipasat/gooddollar#48]
Hello @emipasat!

We want to insert the Google Analytics tag in the Goodmarket to start collect events.

Following the documentation below we want to insert the GA4 tag ( as the Universal one will be disable in 2023)  in the new Goodmarket.

https://www.sharetribe.com/docs/ftw/how-to-set-up-analytics-for-ftw/#google-analytics-4

PS: I followed the instructions and disable the "outbound" click options and also "page changes based on browsers history events" as are mentioned in [github ](https://github.com/sharetribe/ftw-daily/pull/1508) we have some issues with this 2 options"

----

In summary we need to assign the GA4 Tracking ID `G-55T2M7VP3W` in to the environment variable `REACT_APP_GOOGLE_ANALYTICS_ID`

**Comment by emipasat** [comment]
Hi @drptiago 
since G$ is based on an older version of FTW, just changing the environment variable is not enough.

![image](https://user-images.githubusercontent.com/13704492/182006916-fc22d8da-8fb3-4e03-8c33-11e1525d0cae.png)

As per documentation, we need to apply the changes of a pull request.
I'll do that
